src/compiler/checker/lines_0_1000.rs

```diff
@@ -0,0 +1,271 @@
+#![allow(non_upper_case_globals)]
+
+use bitflags::bitflags;
+use std::cell::{RefCell, RefMut};
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use super::create_node_builder;
```

Comment:
For things that are only visible within `src/compiler/checker.ts` I used `super::` imports and `pub(super)` visibility

Reply:
Nice, I've never seen `pub(super)` used.